bitnami/harbor/values-production.yaml

```diff
@@ -692,9 +725,65 @@ registry:
   # Must be a string of 16 chars.
   secret: ""
 
-# Enable Chartmuseum is not supported yet.
 chartmuseum:
-  enabled: false
+  enabled: true
+  replicas: 1
+  port: 8080
+
+  ## Set the use of the Redis cache.
+  useRedisCache: true
+
+  ## Set the absolute URL to access the chartmuseum repository and the
+  ## endpoint where it will be available.
+  absoluteUrl: false
```

Comment:
I'd use `useAbsoluteUrl` instead

Reply:
We tried to use the same names as the original chart. So, all the values are called as in the original one.

Reply:
Okay, it was a suggestion. You can skip it